build/ci/evergreen.yml

```diff
@@ -167,6 +167,29 @@ functions:
           set -Eeou pipefail
 
           curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
+  "install snyk":
+    - command: shell.exec
+      type: setup
+      params:
+        working_dir: src/github.com/mongodb/mongocli/bin
+        script: |
+          set -Eeou pipefail
+
+          curl -sfL https://github.com/snyk/snyk/releases/download/v${snyk_ver}/snyk-linux -o snyk
+          chmod +x snyk
+  "snyk scan":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src/github.com/mongodb/mongocli
+        script: |
+          export SNYK_TOKEN=${snyk_token}
```

Comment:
nit: export before set and also more exports below

Reply:
also, missing ""

Reply:
before because it's a secret, it's an evergreen thing do move these kind of expansions before the `set`s also for expansions we usually don't use quotes(`""`) but I can try it